powerauth-rest-client-spring/src/main/java/com/wultra/security/powerauth/rest/client/PowerAuthFido2RestClient.java

```diff
@@ -84,7 +83,6 @@ public PowerAuthFido2RestClient(String baseUrl, PowerAuthRestClientConfiguration
             if (config.getProxyUsername() != null) {
                 proxyBuilder.username(config.getProxyUsername()).password(config.getProxyPassword());
             }
-            proxyBuilder.build();
```

Comment:
Lime-core API is a little bit confusing. `proxyBuilder.build()` returns `DefaultRestClient.Builder`, which has no effect here.